src/main/webapp/app/shared/components/course-exam-archive-button/course-exam-archive-button.component.html

```diff
@@ -106,6 +107,7 @@ <h4 class="modal-title">
         (delete)="cleanupCourse()"
         [dialogError]="dialogError$"
         deleteConfirmationText="artemisApp.course.delete.typeNameToConfirm"
+        style="margin-right: 0 !important"
```

Comment:
I would prefer a specific CSS selector without `!important` in the SCSS file, if possible

Reply:
Edit: I saw your comment that it's not possible